lib/examples/pssch_ue.c

```diff
@@ -263,13 +263,15 @@ int main(int argc, char** argv)
     printf("Set RX freq: %.6f MHz\n",
            srsran_rf_set_rx_freq(&radio, prog_args.nof_rx_antennas, prog_args.rf_freq) / 1e6);
     printf("Set RX gain: %.1f dB\n", prog_args.rf_gain);
-    int srate = srsran_sampling_freq_hz(cell_sl.nof_prb);
 
+    #define MAX_SRATE_DELTA 2 // Hz allowable delta between requested and actual sample rate
+    /* set sampling frequency */
+    int srate = srsran_sampling_freq_hz(cell_sl.nof_prb);
     if (srate != -1) {
-      printf("Setting sampling rate %.2f MHz\n", (float)srate / 1000000);
+      printf("Setting sampling rate %.2f MHz\n", (float)srate / 1000000);      
```

Comment:
note the white spaces at the end. I'd suggest to setup clang-format in your preferred code editor. Unfortunately, we don't have strict format checks in the public CI

Reply:
Got it - will do